content/show/coder-radio/_index.md

```diff
@@ -12,8 +12,13 @@ active = true
 
 header_image = "/images/shows/coder-radio.png"
 
+
+support_link = "https://jupitersignal.memberful.com/checkout?plan=53334"
+subscribe_link = "https://new.jupiterbroadcasting.com/show/coder-radio/subscribe/"
```

Comment:
Also, since this is hard coded i.e. an absolute link instead of a relative link, this is now broken since #355. Fix is easy..
```suggestion
subscribe_link = "/show/coder-radio/subscribe/"
```

Reply:
Yeap, it will be easy

Reply:
@DronSKM044 I made the changes above in a fashion that can be committed directly to this PR. That's been a handy function in the past, and I wondered if you were familiar with it. It's a nice way for folks to review your contribution, to make a contribution themselves before it's merged, and for you to have those contributions quickly added to this PR.

My question: Do you have the ability to merge my above suggestions via the GH interface? I can merge them on my end, but figured it would be best practice to allow you to review and merge them yourself.

The changes are trivial here, but a good example workflow..

Reply:
I'm not sure I fully understand what you mean. How can I merge above suggestions via the GH interface?

Reply:
![image](https://user-images.githubusercontent.com/7018928/188234181-eb711b35-3fa9-460a-8baa-8a8ac7ef7408.png)